azure-functions-powershell-library/Common/BindingInformation.cs

```diff
@@ -0,0 +1,25 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+namespace Common
+{
+    public class BindingInformation
+    {
+        public enum Directions
+        {
+            In = 0,
+            Out = 1,
+            Inout = 2
+        }
+
+        public int Direction { get; set; } = -1;
```

Comment:
Rather than making this a magic constant, I would rather we expand the `Directions` enumeration above to include something like `Unknown = -1`. Then we can write this as `public Directions Direction { get; set; }` and not have to cast `Directions` into an `int` for the `ExtractBinding` methods.

Reply:
Done